features/order/managing_orders/seeing_order_shipping_taxes.feature

```diff
@@ -0,0 +1,49 @@
+@managing_orders
+Feature: Seeing taxes of an order
+    In order to know shipping and product taxes
+    As an Administrator
+    I want to be able to see taxes in order
+
+    Background:
+        Given the store operates on a single channel
+        And the store ships to "France" and "Australia"
+        And there is a zone "EU" containing all members of the European Union
+        And there is a zone "The Rest of the World" containing all other countries
+        And default currency is "EUR"
+        And default tax zone is "EU"
+        And the store has "EU VAT" tax rate of 23% for "Clothes" within "EU" zone
+        And the store has "Low tax" tax rate of 10% for "Clothes" for the rest of the world
+        And the store has "Shipping EU VAT" tax rate of 23% for "Shipping Services" within "EU" zone
+        And the store has "Shipping Low tax" tax rate of 10% for "Shipping Services" for the rest of the world
+        And the store has a product "PHP T-Shirt" priced at "€80.00"
+        And the store has a product "Symfony2 T-Shirt" priced at "€140.00"
```

Comment:
Only one product belongs to tax category, so it I guess that it could better check taxes on an order